src/components/dashboard/GapByRace.vue

```diff
@@ -1,26 +1,110 @@
 <template>
-  <div class="gap-container">
-    <div>
-      <GapChart
-        :active-stats="activeStats"
-        :expected="expected"
-        :field-data="fieldData"
-      />
+  <!--
+    Depending on the controls, display different charts and layouts
+
+    Implementation:
+    Each child of the outer-container is stacked on top of each other taking up space and his hidden based on the
+    visible class. Conditional rendering would cause resizing of the container which is jarring for the user.
+
+    Use the visible class to determine which is shown rather than v-if / v-else-if / v-else
+  -->
+  <div class="outer-container">
+    <!-- All Residents, display the Gap Chart -->
+    <div
+      class="gap-container"
+      :class="{ visible: focusStat.value === 'total' }"
+    >
+      <div>
+        <GapChart
+          :active-stats="activeStats"
+          :expected="expected"
+          :field-data="fieldData"
+        />
+      </div>
+      <div class="centered">
+        <p class="has-text-centered">
+          In {{ activeCluster.name }}, the largest gap is among
+          {{ activeStats[0]?.name }} residents. Only
```

Comment:
this used to be `minVaxRace`, why no longer? is the first one guaranteed to be the gappiest?

Reply:
I'll bring back `minVaxRace` as a computed value that is set to `activeStats[0]` for readibility